prombench/cmd/prombench/Dockerfile

```diff
@@ -1,15 +1,26 @@
 # NOTE: building this requires it to have the build context of the repository root
```

Comment:
that is not true anymore right?

Reply:
Yes, it's Still true now. If we want to build it locally, have to run `docker build -t prombench:latest -f prombench/cmd/prombench/Dockerfile .` from repository root.